apps/node-host/src/bootstrap.js

```diff
@@ -5,7 +5,9 @@
 
 import express from 'express';
 import * as path from 'path';
-import('node_remote/test');
+import('node_remote/test').then((m) => {
+  console.log(__webpack_require__.federation)
```

Comment:
should return module info as well as any data pushed into __remote_scope__ over proxy. 

Proxy lets us mix global and local metadata about the system over a single local interface. `require.federation`